frontend/src/v5/helpers/form.helper.ts

```diff
@@ -16,6 +16,7 @@
  */
 
```

Comment:
Drawing number should be unique, but I can have more than one drawing using "1" 

https://github.com/3drepo/3drepo.io/assets/43547703/c98cbf2b-ada8-4a0c-a23a-456d752acf1d

Reply:
It seems to be checking the drawing names not numbers when it does the check